languages/woocommerce-gateway-stripe.pot

```diff
@@ -1500,6 +1500,11 @@ msgstr ""
 msgid "Authorize the payment &raquo;"
 msgstr ""
 
+#: vendor/autoload_packages.php:159
+#. translators: %s Name of a PHP Class
+msgid "Not all plugins have loaded yet but we requested the class %s"
+msgstr ""
+
```

Comment:
I think we shouldn't include translations from the vendor folder here. I ran the `makepot` command locally and this was removed from my local files.

We should probably add `vendor` to the `npm makepot` exclusion list to prevent this from creeping in in the future.